src/main/java/com/chessmates/league/LeagueTable.groovy

```diff
@@ -0,0 +1,70 @@
+package com.chessmates.league
+
+import com.chessmates.utility.GameColor
+import org.slf4j.Logger
+
+/**
+ * A simple league table.
+ */
+class LeagueTable {
+
+    private static final Logger logger = org.slf4j.LoggerFactory.getLogger(LeagueTable)
+
+    final winPoints
+    final loosePoints
+
+    private final Map<String, LeagueRow> playerRows = new HashMap()
+
+    LeagueTable(Integer winPoints, Integer loosePoints) {
+        this.winPoints = winPoints
+        this.loosePoints = loosePoints
+    }
+
+    /** Add games to the league table. **/
+    protected void add(List games) {
```

Comment:
This is a detail, but maybe we can accept any collection of games? Does not have to be ordered as a lit.